### DISCUSSION TITLE
Preflight and :host styling

### DISCUSSION THREAD

**Discussion by marcmalerei** [tailwindlabs/tailwindcss#9491] [Help]
``Hi,

i was wondering if there is a way to disable or change a selector in tailwinds preflight. I am having problems styling :host within a component which uses tailwindcss. If the parent component also uses tailwindcss it overwrites some properties like border on the host element. The problematic part is this:

```
*,
::before,
::after {
  box-sizing: border-box; /* 1 */
  border-width: 0; /* 2 */
  border-style: solid; /* 2 */
  border-color: theme('borderColor.DEFAULT', currentColor); /* 2 */
}
```
would be better if it would be like this

```
*:not(:host),
::before,
::after {
  box-sizing: border-box; /* 1 */
  border-width: 0; /* 2 */
  border-style: solid; /* 2 */
  border-color: theme('borderColor.DEFAULT', currentColor); /* 2 */
}
```
Is there a way to do that?

**Comment by wongjn** [comment]
This is from the preflight core plugin. Unfortunately there is no way to modify the CSS output of this with configuration without disabling the preflight core plugin in its entirety and using your own preflight styling with your own modifications.
